### PR TITLE
[TIC-333] Add `/org/{id}/create_saml_connection_link`

### PR DESCRIPTION
**Pull request by itailevi98**
https://propelauth.atlassian.net/browse/TIC-333

### CODE REVIEW

File: src/api/org.ts [hunk 1]
```diff
@@ -272,6 +272,38 @@ export function disallowOrgToSetupSamlConnection(
...
+    integrationApiKey: string,
+    orgId: string,
+    expiresInSeconds?: number
+): Promise<CreateSAMLConnectionLinkResponse> {
+    if (!isValidId(orgId)) {
+        return Promise.reject("Invalid orgId")
+    }
+
+    let endpointPath = `${ENDPOINT_PATH}/${orgId}/create_saml_connection_link`
+    if (expiresInSeconds) {
```
Comment (andrew-propelauth) [inline src/api/org.ts:290]: This should actually be a JSON body parameter, instead of a query parameters.

**Review by andrew-propelauth — CHANGES_REQUESTED** [review]

File: src/api/org.ts (hunk 1 above)
Comment (itailevi98) [reply src/api/org.ts:290]: Ah, I was confused by the naming of the variable in the backend 😅 you're right though, changing now!

Screenshot 2023-10-02 at 5 04 09 PM

Comment (itailevi98) [reply src/api/org.ts:290]: great catch!

**Review by andrew-propelauth — APPROVED** [review]
lgtm!

**Comment by itailevi98** [comment]
Closing this, as it will now go into the `@propelauth/node-apis` library instead.

**Pull request closed without merging**